x-pack/plugins/transform/server/routes/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { RouteDependencies } from '../types';
+
+import { registerPrivilegesRoute } from './api/privileges';
+import { registerTransformsRoutes } from './api/transforms';
+
+import { API_BASE_PATH } from '../../../../legacy/plugins/transform/common/constants';
+
+export const addBasePath = (uri: string): string => API_BASE_PATH + uri;
```

Comment:
Super nit: Any reason template literals aren't being used here?

Reply:
I copied that from the `index_management` plugin, seems they are not using literals if there's no hard-coded strings involved. However, our style guide says `Prefer template strings over string concatenation` so I changed it in 643c4fe.